salt/config/__init__.py

```diff
@@ -1184,7 +1185,7 @@ def _gather_buffer_space():
     'zmq_backlog': 1000,
     'pub_hwm': 1000,
     'auth_mode': 1,
-    'user': 'root',
+    'user': getpass.getuser(),
```

Comment:
I can't find any clear indication of whether or not this is fully supported on Windows. Could you please double-check?